src/features/documentSymbolProvider.ts

```diff
@@ -83,3 +74,9 @@ function toSymbolKind(kind: string): vscode.SymbolKind {
 
     return kinds[kind];
 }
+
+function toVscodeRange(range: protocol.V2.Range):  vscode.Range{
```

Comment:
I think we might have an implementation of this somewhere else already

Reply:
Flavors of this one are present, not exactly this one. However I added this method to that file itself. Now there are three methods that are essentially doing the same thing. Do you think we should refactor a bit here ?